src/linearmixedmodel.jl

```diff
@@ -779,8 +806,9 @@ function σρs(m::LinearMixedModel)
 end
 
 function Base.size(m::LinearMixedModel)
-    n, p = size(first(m.feterms))
-    n, p, sum(size.(m.reterms, 2)), length(m.reterms)
+    n, p = size(fetrm(m))
```

Comment:
`p` is the rank, not just the number of terms, right?

Reply:
Using the size and not the rank here is part of the restructuring around `fixef!`, rightj?